scraper.py

```diff
@@ -0,0 +1,169 @@
+import selenium
+import re
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from webdriver_manager.chrome import ChromeDriverManager
+
+driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))
+# PATH = "/Users/fpw/Desktop/CDP/chromedriver"
+# driver = webdriver.Chrome(PATH)
+
+driver.get("https://atlantacityga.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=3587&Format=Minutes")
```

Comment:
Everything from this point down should be placed into a nice neat function called something like: `parse_page` and it should take in the URL as a parameter.

```python
def parse_page(url: str) -> EventIngestionModel:
    # blah blah blah
    # your code here
```